packages/Config/RectorConfig.php

```diff
@@ -59,7 +59,7 @@ public function disableParallel(): void
         SimpleParameterProvider::setParameter(Option::PARALLEL, false);
     }
 
-    public function parallel(int $seconds = 120, int $maxNumberOfProcess = 16, int $jobSize = 20): void
+    public function parallel(int $seconds = 120, int $maxNumberOfProcess = 16, int $jobSize = 15): void
```

Comment:
reduced the jobsize, which leads to less memory used

Reply:
`config/config.php` need to be updated as well:

https://github.com/rectorphp/rector-src/blob/038f8092eaeeaaf044dbd5685d89bc7636143bbc/config/config.php#L16

could you also update the different between $jobSize vs `MAX_CHUNKS_PER_WORKER` ? Thank you.

Reply:
I dropped the args in this file, as these just reflect the defaults and therefore don't need to be kept in sync

> could you also update the different between $jobSize vs MAX_CHUNKS_PER_WORKER ?

where?

Reply:
here seems ok

↓

https://github.com/rectorphp/rector-src/pull/4965#pullrequestreview-1618817032

Reply:
Looks good :+1: